src/OmniSharp.Cake/Services/RequestHandlers/Navigation/FindSymbolsHandler.cs

```diff
@@ -16,6 +19,17 @@ public FindSymbolsHandler(OmniSharpWorkspace workspace)
         {
         }
 
+        public override Task<QuickFixResponse> Handle(FindSymbolsRequest request)
+        {
+            Func<string, bool> isMatch =
+                candidate => request != null
+                ? candidate.IsValidCompletionFor(request.Filter)
+                : true;
+
+            return  Workspace.CurrentSolution.FindSymbols(isMatch,
```

Comment:
Should the `FindSymbols` extension method just take the file extension? It seems error prone to always remember to specify `StringComparison.OrdinalIgnoreCase`.

Reply:
Good idea.